modern/tests/fixtures/expected.py

```diff
@@ -132,7 +153,20 @@ def expected(request) -> Expected:
     distro = Distro(m.group('distro'), Version(m.group('distro_version')))
     python = Version(env_values.get('PYTHON_VERSION'))
     cmake = Version(env_values.get('CMAKE_VERSION_FULL'))
-    expected = Expected(distro, m.group('username'), m.group('tag'), python, cmake)
+    pkg_config = Version("0.29.1")
+    make = Version("4.1")
+    autoconf = Version("2.69")
+    perl = Version("5.22.1")
+    wget = Version("1.17.1")
+    curl = Version("7.47.0")
+    git = Version("2.7.4")
+    subversion = Version("1.9.3")
+    xzutils = Version("5.1.0alpha")
+    nasm = Version("2.11.08")
```

Comment:
These versions depend on the distro version, right? They are not hardcoded in the Dockerfiles, so they might change. They can even change if the version is modified in the _apt repository_.

Should we hardcode the versions in the Dockerfile? Should we map these versions to distro-version in some external file?

Reply:
We can collect those version online actually. But a local file sounds great too.

Reply:
Thinking again, usually Ubuntu delays package releases, and most of time, they don't change the version, only package version. Let's use a local list, if it starts to hurt, we can adopt a dynamic list from internet.